README.md

```diff
@@ -1 +1,40 @@
-# MicroverseFirstProject
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse 
+
+## Built With
+
+-HTML
+-CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
Great work @rozhi-98 ✋

- [ ] Kindly customize this author with your initials. This is because you are the author of this project. This will make your readme file more professional.